source/js/diva.js

```diff
@@ -939,25 +939,30 @@ module.exports = diva;
         };
 
         /*
-            Given a pageX and pageY value
+         Given clientX and clientY (as could be retrieved from a jQuery event object)
                 returns either the page visible at that (x,y) position or -1 if no page is.
         */
-        this.getPageIndexForPageXYValues = function(clientX, clientY) {
-            if (viewerState.renderer) {
+        this.getPageIndexForPageXYValues = function(clientX, clientY)
+        {
+            if (viewerState.renderer)
+            {
                 var pageHit = viewerState.renderer.getPageHit(clientX, clientY);
-                if (pageHit) {
-                    return pageHit.pg;
+                if (pageHit)
+                {
+                    return pageHit.pageIndex;
                 }
             }
             return -1;
```

Comment:
I'm a bit concerned by the various return values that are being thrown around. Why not return `-1` from `getPageHit` directly? This would mean you wouldn't have to check that `pageHit` here is not null.